subsys/bluetooth/host/l2cap.c

```diff
@@ -2017,7 +2017,7 @@ static int l2cap_chan_le_send_sdu(struct bt_l2cap_le_chan *ch,
 		sent += ret;
 
 		/* If the current buffer has been fully consumed, destroy it */
-		if (ret == rem_len) {
+		if (sent == rem_len) {
```

Comment:
Is `rem` in `rem_len` short for `remaining`?

Reply:
yes